src/Codeception/Module/WebDriver.php

```diff
@@ -798,25 +798,15 @@ public function resizeWindow($width, $height)
     public function seeCookie($cookie, array $params = [])
     {
         $cookies = $this->filterCookies($this->webDriver->manage()->getCookies(), $params);
-        $cookies = array_map(
-            function ($c) {
-                return $c['name'];
-            },
-            $cookies
-        );
+        $cookies = array_column($cookies, 'name');
```

Comment:
I don't know why, but 4 WebDriver tests calling `seeCookie` method fail on Semaphore:

```
There were 4 failures:

---------
1) WebDriverTest: Save session snapshots exclude invalid cookie domains
 Test  tests/web/WebDriverTest.php:testSaveSessionSnapshotsExcludeInvalidCookieDomains
Failed asserting that an array contains 'PHPSESSID'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/web/WebDriverTest.php:718
#3  /home/runner/Codeception/codecept:43

---------
2) WebDriverTest: Cookies with path
 Test  tests/web/WebDriverTest.php:testCookiesWithPath
Failed asserting that an array contains 'cookie'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/unit/Codeception/Module/TestsForWeb.php:813
#3  /home/runner/Codeception/tests/web/WebDriverTest.php:791
#4  /home/runner/Codeception/codecept:43

---------
3) WebDriverTest: Cookies
 Test  tests/web/WebDriverTest.php:testCookies
Failed asserting that an array contains 'test_cookie'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/unit/Codeception/Module/TestsForWeb.php:796
#3  /home/runner/Codeception/tests/web/WebDriverTest.php:777
#4  /home/runner/Codeception/codecept:43

---------
4) WebDriverTest: Session snapshots
 Test  tests/web/WebDriverTest.php:testSessionSnapshots
Failed asserting that an array contains 'PHPSESSID'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/web/WebDriverTest.php:655
#3  /home/runner/Codeception/codecept:43

```

https://semaphoreci.com/codeception/codeception/branches/pull-request-5639/builds/3

Reply:
Reverted 'array_column(...)' changes to see if the changes is causing the build to fail there.